tests/BootstrapperTest.php

```diff
@@ -328,6 +332,82 @@
     expect(File::isDirectory($tenantStoragePath))->toBeFalse();
 });
 
+test('BroadcastTenancyBootstrapper binds TenancyBroadcastManager to BroadcastManager::class and reverts the binding when tenancy is ended', function() {
```

Comment:
```suggestion
test('BroadcastTenancyBootstrapper binds TenancyBroadcastManager to BroadcastManager and reverts the binding when tenancy is ended', function() {
```

The `::class` here is redundant, it's just a way of telling PHP to convert something to a string. But the actual class name is `Namespace\BroadcastManager`, without the `::class`.

Also, we try to avoid using special characters in test names.